tests/integration/module/fixtures/build/shims/shims.test.ts

```diff
@@ -0,0 +1,14 @@
+import { runCli, initBeforeTest } from '../../utils';
+
+initBeforeTest();
+
+describe('shims', () => {
+  const fixtureDir = __dirname;
+  it('shims', async () => {
+    const { success } = await runCli({
+      argv: ['build'],
+      appDirectory: fixtureDir,
+    });
+    expect(success).toBeTruthy();
```

Comment:
Should we check if the result contains the shim?

Reply:
yes, both cjs and esm